src/Testing/CodeHacking/Hacks/CodeHack.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * CodeHack class file.
+ *
+ * @package WooCommerce/Testing
+ */
+
+namespace Automattic\WooCommerce\Testing\CodeHacking\Hacks;
+
+/**
+ * Base class to define Hacks for CodeHacker.
+ *
+ * This class is included for convenience only, any class having a 'public function hack($code, $path)'
+ * can be used as a hack class for CodeHacker.
+ */
+abstract class CodeHack {
+
+	/**
+	 * The hack method to implement.
+	 *
+	 * @param string $code The code to hack.
+	 * @param string $path The path of the file containing the code to hack.
+	 * @return string The hacked code.
+	 */
+	abstract public function hack( $code, $path);
+
+	/**
+	 * Tokenize PHP source code.
+	 *
+	 * @param string $code PHP code to tokenize.
+	 * @return array Tokenized code.
+	 */
+	protected function tokenize( $code ) {
+		//phpcs:ignore PHPCompatibility.FunctionUse.NewFunctionParameters, PHPCompatibility.Constants.NewConstants
+		return PHP_VERSION_ID >= 70000 ? token_get_all( $code, TOKEN_PARSE ) : token_get_all( $code );
```

Comment:
Since WC needs  minimum PHP 7, we can just raise an error if version is less than 7.0

Reply:
Ok, this code was in the original BypassFinals project.

Reply:
feel free to remove it.